selfdrive/ui/translations/main_es.ts

```diff
@@ -804,7 +804,7 @@ Esto puede tardar hasta un minuto.</translation>
     <name>SoftwarePanel</name>
     <message>
         <source>Updates are only downloaded while the car is off.</source>
-        <translation>Apague el auto para descargar actualizaciones.</translation>
+        <translation>Las actualizaciones solo se descargan cuando con el auto apagado.</translation>
```

Comment:
I had to shorten this line a lot to fit in the UI. Can you make this any shorter?

Reply:
Sure, I'll shorten it (also thank you I just realized I forgot to erase something here when translating ops). 

What about "Actualizaciones solo se descargan con el auto apagado." I think this fits in the UI.